### PR TITLE
Create 2023-09-20.md

### PR DESCRIPTION
**Pull request by Mburnett-VA**
Adding in disability end date

### CODE REVIEW

File: content/verification/veteran-verification/release-notes/2023-09-20 [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Additional fields have been added to the disability rating endpoint as follows;
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2023-09-20:1]: Are these additional fields just for v2 of the Veteran Service History and Eligibility API?

File: content/verification/veteran-verification/release-notes/2023-09-20 [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Individual rating end date: (string, $date)
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2023-09-20:3]:
```suggestion
- Individual rating end date, which is the date the service decision expires. An example would be 2019-10-09.
```

File: content/verification/veteran-verification/release-notes/2023-09-20 [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Additional fields have been added to the disability rating endpoint as follows;
+
+Individual rating end date: (string, $date) 
+The date the service decision expires.  
+  Example: 2019-10-09
+Diagnostic type code:  (string)
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2023-09-20:6]:
```suggestion
Diagnostic type code, which is the code that corresponds with the diagnosis. An example would be 1134.
```

File: content/verification/veteran-verification/release-notes/2023-09-20 [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Additional fields have been added to the disability rating endpoint as follows;
+
+Individual rating end date: (string, $date) 
+The date the service decision expires.  
+  Example: 2019-10-09
+Diagnostic type code:  (string) 
+Type code for the diagnosis.  
+  Example:  1134
+Diagnostic text:  (string)
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2023-09-20:9]:
```suggestion
Diagnostic text, which is a description of the diagnosis. An example would be bilateral hearing loss.
```

**Review by joeabraun — CHANGES_REQUESTED** [review]
Left some questions and suggestions

File: content/verification/veteran-verification/release-notes/2023-09-20 (hunk 1 above)
Comment (Mburnett-VA) [reply content/verification/veteran-verification/release-notes/2023-09-20:1]: Yes, just for V2

Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2023-09-20:1]:
```suggestion
We've added some additional fields to the disability rating endpoint for v2 of the Veteran Service History and Eligibility API:
```

File: content/verification/veteran-verification/release-notes/2023-09-20 [hunk 5]
```diff
@@ -0,0 +1,11 @@
+Additional fields have been added to the disability rating endpoint as follows;
+
+- Individual rating end date, which is the date the service decision expires. An example would be 2019-10-09.
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2023-09-20:3]:
```suggestion
- Individual rating end date, which is the date the service decision expires. An example would be 2019-10-09.
```

File: content/verification/veteran-verification/release-notes/2023-09-20 [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+The date the service decision expires.
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2023-09-20:4]:
```suggestion
```

File: content/verification/veteran-verification/release-notes/2023-09-20 [hunk 7]
```diff
@@ -0,0 +1,11 @@
+Additional fields have been added to the disability rating endpoint as follows;
+
+- Individual rating end date, which is the date the service decision expires. An example would be 2019-10-09.
+The date the service decision expires.  
+  Example: 2019-10-09
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2023-09-20:5]:
```suggestion
```

File: content/verification/veteran-verification/release-notes/2023-09-20 [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+Diagnostic type code, which is the code that corresponds with the diagnosis. An example would be 1134.
+Type code for the diagnosis.
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2023-09-20:7]:
```suggestion
```

File: content/verification/veteran-verification/release-notes/2023-09-20 [hunk 9]
```diff
@@ -0,0 +1,11 @@
+Additional fields have been added to the disability rating endpoint as follows;
+
+- Individual rating end date, which is the date the service decision expires. An example would be 2019-10-09.
+The date the service decision expires.  
+  Example: 2019-10-09
+Diagnostic type code, which is the code that corresponds with the diagnosis. An example would be 1134.
+Type code for the diagnosis.  
+  Example:  1134
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2023-09-20:8]:
```suggestion
```

**Review by joeabraun — CHANGES_REQUESTED** [review]
Requested some changes

File: content/verification/veteran-verification/release-notes/2023-09-20 [hunk 10]
```diff
@@ -0,0 +1,7 @@
+We've added some additional fields to the disability rating endpoint for v2 of the Veteran Service History and Eligibility API:
+
+- Individual rating end date, which is the date the service decision expires. An example would be 2019-10-09.
+Diagnostic type code, which is the code that corresponds with the diagnosis. An example would be 1134.
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2023-09-20:4]:
```suggestion
- Diagnostic type code, which is the code that corresponds with the diagnosis. An example would be 1134.
```

File: content/verification/veteran-verification/release-notes/2023-09-20 [hunk 11]
```diff
@@ -0,0 +1,7 @@
...
+Diagnostic text, which is a description of the diagnosis. An example would be bilateral hearing loss.
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2023-09-20:5]:
```suggestion
- Diagnostic text, which is a description of the diagnosis. An example would be bilateral hearing loss.
```

**Review by joeabraun — CHANGES_REQUESTED** [review]
Added a couple more changes.

**Review by joeabraun — CHANGES_REQUESTED** [review]
Some final changes

**Pull request merged**